scripts/webidl/convert.js

```diff
@@ -51,3 +51,4 @@ doConversion(path.resolve(__dirname, "../../lib/jsdom/living/events")).done();
 doConversion(path.resolve(__dirname, "../../lib/jsdom/living/attributes")).done();
 doConversion(path.resolve(__dirname, "../../lib/jsdom/living/window")).done();
 doConversion(path.resolve(__dirname, "../../lib/jsdom/living/nodes")).done();
+doConversion(path.resolve(__dirname, "../../lib/jsdom/living/traversal")).done();
```

Comment:
OK, so, traversal needs to happen before nodes, since Document depends on TreeWalker.

This needs to be redone as something like

```js
doConversion(...traversal")
  .then(() => doConversion(...events"))
  .then(() => doConversion(...attributes"))
  .then(() => doConversion(...window"))
  .then(() => doConversion(...nodes"))
  .done();
```